projects/fine_tuning/testing/config.yaml

```diff
@@ -730,7 +730,8 @@ tests:
       image: quay.io/rhoai/ray:2.35.0-py311-cu121-torch24-fa26
     ilab:
       enabled: false
-      image: registry.redhat.io/rhelai1/instructlab-nvidia-rhel9@sha256:b3dc9af0244aa6b84e6c3ef53e714a316daaefaae67e28de397cd71ee4b2ac7e
+#      image: registry.redhat.io/rhelai1/instructlab-nvidia-rhel9@sha256:b3dc9af0244aa6b84e6c3ef53e714a316daaefaae67e28de397cd71ee4b2ac7e
+      image: registry.redhat.io/rhelai1/instructlab-nvidia-rhel9@sha256:525ab53de3829cac1a9aabb73194f49e22da8fdcf12a01c56ece961300cdab0d
```

Comment:
can you remove the 1.2 image, and add `# instructlab 1.3` comment at the end of the `image:` line?